src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -136,7 +141,7 @@ private void HandleMessage(Message message, Request request, IIssueConsumer cons
             // returned. This doesn't cause a crash; all active taggers will have been detached from the
             // TextBufferIssueTracker when the file was closed, but the TextBufferIssueTracker will
             // still exist and handle the call.
-            consumer.Accept(request.File, new[] { issue });
+            consumer?.Accept(request.File, new[] { issue });
```

Comment:
If the consumer is null then there's no point in doing any of the work in this method (or even in calling the method).